source/_patterns/00-protons/non-printing/_colors.scss

```diff
@@ -1,9 +1,4 @@
-// Custom colors
+// Custom colors variables and mixins
 
-// Bootstrap overrides
-// Add colors to $theme-colors var here. Find other vars to override at ~bootstrap/scss/_variables.scss
-// Be sure to add new colors in quotes.
-$theme-colors: (
-  'test': pink,
-  'new': green,
-);
+$color-blerp: pink;
+$color-derp: green;
```

Comment:
this implies we need to add colors vars in two places now--- initially as a variable here, and then referenced again in the _bootstrap-overrides to get all the extra goodies that bootstrap does with the $theme-colors map (e.g. generate css vars, color helper classes). i'm not opposed but that could get annoying / provide another point of failure if people aren't aware

Reply:
In a way I prefer this for this reason:

Our custom colors are our own, totally independent of Bootstrap. We then give them to Bootstrap specifically.

This slowly lets us move down the path to renaming `bootstrap-printing` and `bootstrap-overrides` to `vendor-printing` and `vendor-overrides`.